ramalama/common.py

```diff
@@ -1,17 +1,38 @@
-"""ramalama common module."""
```

Comment:
I would prefer to avoid putting these licenses headers everywhere, it's a pain, you have to remember to update the year, etc. One license file is sufficient.

Reply:
Oh no,  usually is good to have header license. Linux kernel and others use it very often.

Reply:
Linux kernel do a lot of things, I don't think we have to copy them though. PyTorch don't do this and I'm sure we could find many projects that do or don't.

It's not worth the maintenance, we are licensed . We don't do this for any other python file in the project.

Reply:
I think Linux kernel community doesn't even recommend this anymore themselves, I think they recommend one-liner SPDX (which I don't think we need either).

It's just unnecessary process and maintenance IMO, unless we are legally advised to do it, I'd rather not.